src/widgets/Markdown/markdown-it-tasklist.ts

```diff
@@ -0,0 +1,79 @@
+// SPDX-FileCopyrightText: © 2016 Revin Guillen
```

Comment:
Can you please add `// SPDX-License-Identifier: MIT` on the following line?

Reply:
Yep. Sorry I forgot about that